generators/server/templates/src/main/java/package/web/filter/OAuth2ReactiveRefreshTokensWebFilter.java.ejs

```diff
@@ -46,6 +46,7 @@ public class OAuth2ReactiveRefreshTokensWebFilter implements WebFilter {
             .filter(principal -> principal instanceof OAuth2AuthenticationToken)
             .cast(OAuth2AuthenticationToken.class)
             .flatMap(authentication -> authorizedClient(exchange, authentication))
+            .onErrorResume(e -> Mono.empty())
```

Comment:
It should redirect to the oauth2 login here.
If the token is expired, an error is returned.